docs/execution-providers/MIGraphX-ExecutionProvider.md

```diff
@@ -84,8 +93,62 @@ You can check [here](https://github.com/scxiao/ort_test/tree/master/python/run_o
 model on either the CPU or MIGraphX Execution Provider.
 
 
-## Configuration Options
-MIGraphX providers an environment variable ORT_MIGRAPHX_FP16_ENABLE to enable the FP16 mode.
+## Session Variables
+
+These flags can be invoked via the Provider Options struct [link](https://github.com/ROCm/onnxruntime/blob/rocm6.4_internal_testing/include/onnxruntime/core/session/onnxruntime_c_api.h#L615) when creating an Onnxruntime Session Object and invoking the MIGraphXExecutionProvider 
+
+Items are added as a python dictionary when invoking the MIGraphX execution provider when using python
+
+|Session Option Flag | Values | Description|
+|---|---|---|
+| device_id | INT | Select the device ID specified for the session run (default will be device 0) | 
+| migraphx_fp16_enable | 1 or 0 | Enable FP16 quantization mode via the MIGraphX API of the input model. |
+| migraphx_int8_enable | 1 or 0 | Enable int8 static quantization mode of the input model via the MIGraphX API.\n Requires calibration table path vars to be set (migraphx_int8_calibration_table_name=<valid path>).|
```

Comment:
\n not working. See preview: https://github.com/microsoft/onnxruntime/blob/4466b403a1d91be8feb193393ccb0ff111bcb1de/docs/execution-providers/MIGraphX-ExecutionProvider.md

Reply:
Thats fine I can remove this